### ISSUE TITLE
next_prerelease

### ISSUE THREAD

**Issue by gsakkis** [rbarrois/python-semanticversion#93]
I'd like to suggest a new `Version.next_prerelease` method, unless there is a reason it hasn't been implemented(?).

**Comment by rbarrois** [comment]
Hi!

The main reason would be that nobody suggested it yet ;)
However, I'm not sure how to implement it: how would we compute "the" next pre-release value?

**Comment by gsakkis** [comment]
Indeed, it turns out [it's not quite clear](https://github.com/python-semver/python-semver/issues/221) how the next release from/to a pre-release should be determined; feel free to chime in the discussion :)

**Comment by rbarrois** [comment]
Yeah, basically such a feature would require a pre-release versioning scheme definition (i.e: what comes after `1.0.1-a1`: is it `1.0.1-a2`, `1.0.1-a1.1`, `1.0.1-b1`, ...?).

I'm not aware of any specification for such schemes, so I believe that this should rather be left to users of the library to define.

**Comment by rbarrois** [comment]
I'm gonna go ahead and close this issue, as we both agree it's a complicated world out there :D

For future readers of this issue: if an official definition for a "next" prerelease version arises, please reopen this issue!
